app/parse.py

```diff
@@ -8,5 +13,47 @@ class Course:
     duration: str
 
 
+def parse_single_course(course: BeautifulSoup) -> Course:
+    return Course(
+        name=course.select_one(
+            "h1[data-qa='profession-title']"
+        ).text.split(":")[0],
```

Comment:
The CSS selector used here assumes a specific structure of the HTML. If the website changes, this selector might break. Consider adding error handling to manage cases where the selector does not find the expected element.